src/app/item-page/media-viewer/media-viewer-video/media-viewer-video.component.ts

```diff
@@ -26,9 +28,39 @@ export class MediaViewerVideoComponent implements OnInit {
 
   ngOnInit() {
     this.isCollapsed = false;
-    this.filteredMedias = this.medias.filter(
-      (media) => media.format === 'audio' || media.format === 'video'
-    );
+    this.filteredMedias = this.medias.filter((media) => media.format === 'audio' || media.format === 'video');
+  }
+
+  /**
+   * This method check if there is caption file for the media
+   * The caption file name is the media name plus "-" following two letter
+   * language code and .vtt suffix
+   *
+   * html5 video only support WEBVTT format
+   *
+   * Two letter language code reference
+   * https://www.w3schools.com/tags/ref_language_codes.asp
+   */
+  getMediaCap(name: string): CaptionInfo[] {
+    let filteredCapMedias: MediaViewerItem[];
+    let capInfos: CaptionInfo[] = [];
+    filteredCapMedias = this.medias
+       .filter((media) => media.minetype === 'text/vtt')
```

Comment:
We may want to add the 'text/vtt' mime type to the DSpace registry by default (this would require a separate backend PR).  That way it's easier for others to reuse this code.  For example, if someone forgets to add this mimeType & then uploads a VTT file, captioning won't work.